main.py

```diff
@@ -143,9 +144,9 @@ def run(filename, rule_type=RuleType.ALL, with_console_output=False, execution_m
                                 if match:
                                     instance_id = '#' + match.group(1)
                                     results = re.sub(pattern, '', str(results))
-                                else:
-                                    instance_id = "-"
-                                yield f"{feature_name}/{scenario_name}.v{version}", f"{remote}/blob/{shas[0]}/{feature_file}", f"{step_name}", instance_id, results
+                                yield f"{feature_name}/{scenario_name}.v{version}", f"{remote}/blob/{shas[0]}/{feature_file}", f"{scenario_name}", f"{step_name}", instance_id, results
+                            elif item['activated']:
```

Comment:
I'm a bit hesitant here on declaring victory at this point and yielding "Rule passed", because there can be more `then` steps in the scenario or more scenarios. Granted, in none of the 000-series we have multiple then steps or multiple scenarios, but I think the logic here needs to be match the more general case.

We can solve this in two ways:
- Don't immediately yield RulePassed when item['activated'], but do so at the end of the outer for loop when making sure that we really never got any `results`.
- Reverse the logic and yield `"Rule activated"` whenever we see `item['activated']` and adapt the logic in test_main.